src/contacts.cs

```diff
@@ -545,6 +547,31 @@ interface CNContainer : NSCopying, NSSecureCoding {
 
 		[Export ("type", ArgumentSemantic.Assign)]
 		CNContainerType ContainerType { get; }
+
+		[Static]
```

Comment:
add a #region over the members that say it comes from a category. It makes it easier to to confuse them with normal static methods